lib/susedistribution.pm

```diff
@@ -503,7 +503,6 @@ sub activate_console {
             $nr = get_root_console_tty if ($name eq 'root');
             $nr = 5 if ($name eq 'log');
             my @tags = ("tty$nr-selected", "text-logged-in-$user");
-            push(@tags, 'text-login') unless $args{ensure_tty_selected};
```

Comment:
If you remove this then you should remove the block in line R509-R525 as well.

Both https://w3.nue.suse.com/~okurz/openqa_opensuse_org_status.html as well as https://w3.nue.suse.com/~okurz/openqa_suse_de_status.html reference existing tests which hit this code branch so I guess you can start with these scenarios first, see why tty$nr-selected does not match there, make sure they do and then remove the text-login needle. As I saw in another comment by you that you identified test suites which use text-login in another place maybe it makes sense to rename the used tag either here or in the other places to separate the issues